### ISSUE TITLE
Ugly history

### ISSUE THREAD

**Issue by hannesa2** [googlesamples/mlkit#203]
This is just a feedback.
It's really hard to follow your changes by your ugly git history without meaningful commit message

I at least squashed all, to see changes at once. But it's still hard to have as result one huge commit

<img width="646" alt="image" src="https://user-images.githubusercontent.com/3314607/101978017-245d2100-3c52-11eb-9001-6319761f2449.png">

**Comment by chrisito** [comment]
Thanks for reporting, annoying indeed. We fixed this for the Android sample apps going forward. For iOS samples we are working on it.

**Comment by bcdj** [comment]
This is fixed for future iOS sample app changes as well.

**Comment by hannesa2** [comment]
This is a step in the right direction 
<img width="579" alt="image" src="https://user-images.githubusercontent.com/3314607/109810818-4e5b2980-7c2a-11eb-86c0-46cbcbe71f4d.png">

It would be nice, if you add in short the change context in **very first line**.
Thank you !

**Comment by hannesa2** [comment]
@bcdj Maybe you fixed something, but I still can only guess on commit message what's changed 
<img width="939" alt="image" src="https://user-images.githubusercontent.com/3314607/113513689-3d743f80-956b-11eb-9194-6d286736bc76.png">
